modDbImpl/src/org/aion/db/impl/h2/H2MVMap.java

```diff
@@ -148,7 +148,8 @@ public boolean open() {
         } catch (Exception e) {
             if (e instanceof NullPointerException) {
                 LOG.error("Failed to open the database " + this.toString()
-                        + ". A probable cause is that the H2 database cannot access the file path.", e);
+                    + ". A probable cause is that the H2 database cannot access the file path. "
+                    + "Check that you do not have two instances running on the same database.", e);
```

Comment:
Minor: May be change line 152 to "Check if you have two instances running" instead of "Check that you do not" and everywhere with the same statement.